examples/files.json

```diff
@@ -354,6 +354,7 @@
 		"webgpu_materials_arrays",
 		"webgpu_materials_basic",
 		"webgpu_materials_displacementmap",
+		"webgpu_materials_envmaps_bpcem",
```

Comment:
How about a different name, for clarity?

```js
webgpu_materials_envmaps_box_projected
```
Similar to
```js
webgl_materials_envmaps_ground_projected
```
(The latter is currently missing an underscore.)

Reply:
> webgl_materials_envmaps_ground_projected

I think this should be `webgl_skybox_ground_projected`. It's not really about environment mapping...

Reply:
Whatever you prefer is OK with me... I thought the acronym may be unfamiliar.